pipeline/scraper/npm_query.py

```diff
@@ -34,17 +24,31 @@ def get_package_metadata(dependency):
 
     entry['name'] = dependency
 
-    # Get average downloads
+    # Get downloads
     try:
-        res = requests.get(f'{NPM_LAST_MONTH_DOWNLOADS_META_API_URL}/{dependency_name}')
-        res_json = res.json()
-
-        entry['downloads_last_month'] = 0
-        if res_json.get('downloads'):
-            entry['downloads_last_month'] = get_average_monthly_donwloads(res_json.get('downloads'))
+        now_date = datetime.datetime.now()
+        last_month_start = month_delta(now_date, 1).strftime("%Y-%m-%d")
+        last_month_end = now_date.strftime("%Y-%m-%d")
+        month_last_year_start = month_delta(now_date, 13).strftime("%Y-%m-%d")
+        month_last_year_end = month_delta(now_date, 12).strftime("%Y-%m-%d")
+        last_month_url = f'{NPM_DOWNLOADS_API_URL}/{last_month_start}:{last_month_end}/{dependency_name}'
+        month_last_year_url = f'{NPM_DOWNLOADS_API_URL}/{month_last_year_start}:{month_last_year_end}/{dependency_name}'
+        last_month_downloads_json = requests.get(last_month_url).json()
+        month_last_year_downloads_json = requests.get(month_last_year_url).json()
+
+        # Get monthly downloads over the past month
+        entry['monthly_downloads_last_month'] = 0
+        if last_month_downloads_json.get('downloads'):
+            entry['monthly_downloads_last_month'] = last_month_downloads_json['downloads']
+
+        # Get monthly downloads for the month a year ago
+        entry['monthly_downloads_a_year_ago'] = 0
+        if month_last_year_downloads_json.get('downloads'):
+            entry['monthly_downloads_a_year_ago'] = month_last_year_downloads_json['downloads']
```

Comment:
not everything needs to be in try block since exception expected only on request.get() seems like